### DISCUSSION TITLE
Workflow issue? (PR labeler does not work)

### DISCUSSION THREAD

**Discussion by ejeschke** [ejeschke/ginga#922] [Developers]
@pllim, I got the [following issue](https://github.com/ejeschke/ginga/actions/runs/465415478) for a PR I submitted.  Not sure if this is related to recent CI changes.  Any idea?

**Comment by pllim** [comment]
Ops! Not sure how that happened. I fixed it in https://github.com/ejeschke/ginga/commit/ee083502e79f066553db6b5784351b8b2b7f921f . I opted to do it straight on `master` branch, as the fix is trivial. Hope you don't mind. It should work for the next PR that is opened. If not, please let me know!

**Reply by ejeschke** [reply]
No problem. Thanks and Happy New Year, @pllim!

**Reply by pllim** [reply]
Happy New Year!
